dhis-2/dhis-web-api/src/main/java/org/hisp/dhis/webapi/controller/ConfigurationController.java

```diff
@@ -157,6 +157,28 @@ public void setFeedbackRecipients( @RequestBody String uid )
         configurationService.setConfiguration( config );
     }
 
+    @PreAuthorize( "hasRole('ALL') or hasRole('F_SYSTEM_SETTING')" )
+    @PostMapping( "/systemUpdateNotificationRecipients" )
+    @ResponseStatus( HttpStatus.NO_CONTENT )
+    public void setSystemUpdateNotificationRecipients( @RequestBody String uid )
+        throws NotFoundException
+    {
+        uid = trim( uid );
```

Comment:
This feels unnecessary - otherwise we would need to trim all string API parameters - also wonder why we do not use a request parameter instead. Its name would communicate that we want a user group ID.

Reply:
I was copying the setFeedbackRecipient method. This trim method is used extensively in the controller, I have not investigated why. This also seems unnecessary to me, but since this is used all over the class, it feels like there might be legitimate use, maybe the UI app/client using this has some special way of doing requests. I suggest keeping use the same pattern, otherwise we have to coordinate with front-end to investigate this. This feels like a separate issue.